Makefile

```diff
@@ -158,6 +158,11 @@ lint-pylint: ## lint back-end python sources with pylint only on changed files f
 	bin/pylint --diff-only=origin/main
 .PHONY: lint-pylint
 
+lint-pylint-all: ## lint back-end python sources with pylint
```

Comment:
IMO the subject of your commit is `makefile` and not `backend`. I though you was installing a new dependency